### PR TITLE
Readme clarity of text

### PR DESCRIPTION
**Pull request by playwithreality**
Needs checking

### CODE REVIEW

**Comment by okartau** [comment]
add to whitelist

File: README.MD [hunk 1]
```diff
@@ -166,20 +166,29 @@ The above diagram shows the steps when setting up the BBB and booting it:
 4. BBB executes the kernel, which is the bootsystem core and is needed to be able to boot the rest of the system via NFS.
 5. Boot the rest of the system via NFS.
 
+## 2.3 Setting up BeagleBone Black and DUT filesystems on the host PC
 
+Here you will set up the filesystems for BBB and DUT support image. The filesystems will be made available via NFS which is set up after this part in sections from 2.4 to 2.5.
```
Comment (Arthil) [inline README.MD]: Here WE will set up (let's keep the same format), which will be setup in the sections 2.4-2.5

File: README.MD [hunk 2]
```diff
@@ -8,7 +8,7 @@
 2. [Creating DAFT setup](#2-creating-daft-setup)
     * 2.1 [Required hardware](#21-required-hardware)
     * 2.2 [BBB booting](#22-bbb-booting)
-    * 2.3 [Setting up BeagleBone Black filesystem on the host PC](#23-setting-up-beaglebone-black-filesystem-on-the-host-pc)
+    * 2.3 [Setting up BeagleBone Black and DUT filesystems on the host PC](#23-setting-up-beaglebone-black-and-dut-filesystems-on-the-host-pc)
```
Comment (Arthil) [inline README.MD]: This is missing the sections 2.3.1 and 2.3.2

File: README.MD [hunk 3]
```diff
@@ -535,7 +548,7 @@ _gpio_cutter_on_ and _gpio_cutter_off_ values:
 <pre>
 sudo vim /daft/bbb_fs/etc/aft/devices/platform.cfg
 </pre>
-<pre>
+<pre>  readme_clarity_of_text
```
Comment (Arthil) [inline README.MD:551]: This shouldn't probably be here.

**Pull request closed without merging**